content/applications/websites/elearning.rst

```diff
@@ -0,0 +1,258 @@
+     enroll process to subscribe to the course. This message appears on your website under the
+     course title;
+   - :guilabel:`On Payment`: the :guilabel:`Paid Courses` features must be enabled in your settings
+     to get this option. If you select :guilabel:`On Payment`, you must add a :guilabel:`Product`
+     for your course. Select an existing one or create a new one. It can be updated by clicking on
+     the green arrow that redirects you to the :doc:`../sales/sales` app to finalize the
+     configuration.
+
+.. _elearning/display:
+
+Display
+*******
+
+- :guilabel:`Training`: the course content appears as a training program, and the courses must be
+  taken in the proposed order.
+- :guilabel:`Documentation`: the content is available any order. If you chose this option, you can
+  select the :guilabel:`Featured Content` on the course homepage.
+
+.. _elearning/karma:
+
+Karma tab
+~~~~~~~~~
+
+This tab is about gamification to make e-learning fun and interactive.
+
+In the :guilabel:`Rewards` section, set up the rewards you want to grant your students when they
+:guilabel:`Review` or :guilabel:`Finish` a course.
+
+In the :guilabel:`Access Rights` section, define the karma needed to :guilabel:`Add Review`,
+:guilabel:`Add Comment`, or :guilabel:`Vote` for the course.
+
+.. note::
+   From your course, use the smart button :guilabel:`Contact attendees` to reach people who are
+   already part of the course.
+
+.. _elearning/create-content:
+
+Content
+-------
+
+Manage your content by going to :menuselection:`eLearning --> Courses --> Contents`. Click
+:guilabel:`New` to create content. Add your :guilabel:`Content Title`, and if you want
+:ref:`Tags <elearning/tags>`, then fill in the related information among the different tabs.
+
+.. image:: elearning/elearning-content-tab.png
+   :align: center
+   :alt: Create your content.
+
+Document tab
+~~~~~~~~~~~~
+
+- :guilabel:`Course`: select the course your content belongs to;
+- :guilabel:`Content Type`: select your type of content;
+- :guilabel:`Responsible`: add a responsible person for your content;
+- :guilabel:`Duration`: indicate the duration required to complete the course;
+- :guilabel:`Allow Download`: allow users to download the content of the slide. This option is only
+  visible when the content is a document;
+- :guilabel:`Allow Preview`: the course is accessible by anyone, users don’t need to join the
+  channel to access the content of the course;
+- :guilabel:`# of Public Views`: it represents the views of the non-attending participants;
+- :guilabel:`# Total Views`: it represents the total of the public views and the attendees views.
+
+Description tab
+~~~~~~~~~~~~~~~
+
+You can add a description of your content. It appears front end in the :guilabel:`About` section of
+your course content.
+
+Additional resources tab
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+Click :guilabel:`Add a line` to add links or resources that benefit your participants. They appear
```

Comment:
Click :guilabel:`Add a line` to add links or resources that support your participants' learning.